Tests/TuistCoreTests/MetadataProviders/XCFrameworkMetadataProviderTests.swift

```diff
@@ -272,4 +239,48 @@ final class XCFrameworkMetadataProviderTests: TuistTestCase {
         // Then
         XCTAssertEqual(metadata.macroPath, macroPaths.sorted().first)
     }
+
+    func test_loadMetadataXCFrameworkDylibBinary() throws {
+        // Given
+        let frameworkPath = fixturePath(path: try RelativePath(validating: "MyMath.xcframework"))
+
+        // When
+        let metadata = try subject.loadMetadata(at: frameworkPath, status: .required)
+
+        // Then
+        let expectedInfoPlist = XCFrameworkInfoPlist(libraries: [
+            .init(
```

Comment:
Nitpick: I'd use the full name of the symbol instead of `.init`, which adds some overhead to the compiler when resolving the symbol associated to this.

Reply:
Nice point, this is my style as well. Here is for matching the existing style. If we prefer the explicit type name style, I can update the entire `XCFrameworkMetadataProviderTests.swift` file.